backend/db.js

```diff
@@ -0,0 +1,14 @@
+export const EXERCISES = [
```

Comment:
I would write `exercises` instead of `EXERCISES`. It is recommend to write normal variables in camelcase.